src/api/app/models/project.rb

```diff
@@ -78,7 +78,6 @@ def autocomplete(search)
   has_many :target_of_bs_requests, through: :target_of_bs_request_actions, source: :bs_request
 
   has_one :staging, class_name: 'StagingWorkflow', inverse_of: :project
-  belongs_to :staging_workflow, inverse_of: :staging_projects
   has_many :staged_requests, class_name: 'BsRequest', foreign_key: 'staging_project_id', dependent: :nullify
```

Comment:
I think this association is not needed anymore because it was moved to `staging_project`. BTW, `dependent: :nullify` is not included in the assocation defined in [staging_project](https://github.com/openSUSE/open-build-service/pull/6224/files#diff-61726d0667701e350f427c3907255462R3)

Reply:
I had to re-add this because it's used in some existing code. Will check how this can be resolved